src/main/java/hudson/plugins/plot/PlotCategoryDataset.java

```diff
@@ -38,24 +33,24 @@ public DataElement(Number n, String u) {
     }
 
     /** The row keys */
-    private List<Comparable> rowKeys;
+    private transient List<Comparable> rowKeys;
 
     /** The column keys */
-    private List<Comparable> columnKeys;
+    private transient List<Comparable> columnKeys;
 
     /** The row data */
-    private List<Map<Comparable, DataElement>> data;
+    private transient List<Map<Comparable, DataElement>> data;
 
     /** The max number of builds to plot */
-    private int maxColumns;
+    private transient int maxColumns;
 
     /**
      * Creates a new empty instance.
      */
     public PlotCategoryDataset() {
-        this.rowKeys = new ArrayList<Comparable>();
-        this.columnKeys = new ArrayList<Comparable>();
-        this.data = new ArrayList<Map<Comparable, DataElement>>();
+        this.rowKeys = new ArrayList<>();
```

Comment:
Why?

Reply:
Same as in https://github.com/jenkinsci/plot-plugin/pull/32#discussion_r137703888